nomad/api/base.py

```diff
@@ -33,22 +33,24 @@ def _required_namespace(endpoint):
                                 "deployments",
                                 "acl"
                              ]
+        # split 0 -> Api Version
+        # split 1 -> Working Endpoint
         endpoint_split = endpoint.split("/")
-        return endpoint_split[0] in required_namespace
+        try:
+            required = endpoint_split[1] in required_namespace
```

Comment:
lets name the index instead of just have the number or comments; something like
```
ENDPOINT_NAME=1
...
required = endpoint_split[ENDPOINT_NAME] in required_namespace
```

Reply:
Hi again @jrxFive . Change done. Pushed on my branch and the test will be executed